packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, describe, test, beforeEach, afterEach } from '@jest/globals'
+import { TestUtils, type StreamState, type IpfsApi } from '@ceramicnetwork/common'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { createIPFS, swarmConnect, withFleet } from '@ceramicnetwork/ipfs-daemon'
+import type { Ceramic } from '../ceramic.js'
+import { createCeramic as vanillaCreateCeramic } from './create-ceramic.js'
+import { Model, ModelDefinition } from '@ceramicnetwork/stream-model'
+import { take } from 'rxjs'
+import { Document } from '../feed.js'
+
+function createCeramic(
+  ipfs: IpfsApi,
+  anchorOnRequest = false,
+  streamCacheLimit = 100
+): Promise<Ceramic> {
+  return vanillaCreateCeramic(ipfs, {
+    anchorOnRequest,
+    streamCacheLimit,
+  })
+}
+
+describe('Ceramic feed', () => {
+  let ipfs: IpfsApi
+  let ceramic: Ceramic
+  beforeEach(async () => {
+    ipfs = await createIPFS()
+    ceramic = await createCeramic(ipfs)
```

Comment:
some tests don't use this ceramic instance at all since they make their own.  Can we do one of:
1. *always* leave the tests to create their own Ceramic
2.  change this setup to happen `beforeAll` instead of `beforeEach` so we aren't paying the cost of setting this up and tearing it down between each test?
3. set up 2 ceramic nodes in the test harness and use those in the tests, do no node setup within the body of any tests.

My personal preference is probably for 3, but I'm fine with any approach